srcpkgs/x2goserver/files/x2goserver/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+/usr/sbin/x2gocleansessions
+pid=$(cat '/run/x2goserver.pid')
+trap "kill ${pid}" EXIT
+tail --pid="${pid}" -f /dev/null
```

Comment:
I don’t know how to fix it, but this entire script so against `runit` philosophy.

Reply:
Yep, it's quite a dirty workaround because x2go automatically forks into the background.